scripts/setup/ubuntu/install_deps.sh

```diff
@@ -52,3 +52,9 @@ PYTHON_DEPS=(
 )
 
 python3 -m pip install "${PYTHON_DEPS[@]}"
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
```

Comment:
could you add a comment above this line explaining what it does if possible?

Reply:
Done.